setup.py

```diff
@@ -11,18 +11,23 @@
 
 setup(
     name="nexmo",
-    version="2.5.1",
+    version="2.5.2",
```

Comment:
Is this going to be `2.5.2` or `2.5.1`?

Reply:
2.5.2

Reply:
Wait, why is this going to be 2.5.2? We haven't released 2.5.1 yet

Reply:
For additional context from PMs - 2.5.1 was accidently packaged and pushed to Pypi for a brief moment, but Pypi won't allow versions to be reused.